coreclient/src/clients/mod.rs

```diff
@@ -1134,6 +1134,39 @@ impl CoreUser {
         Ok(count)
     }
 
+    /// Updates the client's push token on the QS.
+    pub async fn update_push_token(&self, push_token: Option<PushToken>) -> Result<()> {
```

Comment:
Why does this take an optional push token? This doesn't seem to do anything if there isn't a push token given as input.